packages/gatsby-link/src/internal.d.ts

```diff
@@ -0,0 +1,29 @@
+export {}
+
+declare global {
+  const __BASE_PATH__: string | undefined
+  const __PATH_PREFIX__: string
+
+  const ___loader: {
+    enqueue: (...args: unknown[]) => void
+    hovering: (arg: unknown) => void
+  }
+
+  interface Window {
+    ___navigate(arg: unknown, options: unknown): void
+    ___push(arg: unknown): void
+    ___replace(arg: unknown): void
```

Comment:
why are these considered `unknown`? We know these types don't we?

Reply:
Yeah honestly I'm not sure (and I don't know where they come from), so pointers would be appreciated!